packages/editor/src/components/commands/index.js

```diff
@@ -290,6 +302,28 @@ function useEditedEntityContextualCommands() {
 			},
 		} );
 	}
+	if ( isViewable ) {
+		const isPage = postType === 'page';
+		let label;
+
+		if ( status === 'publish' ) {
+			label = isPage ? __( 'View page' ) : __( 'View post' );
+		} else {
+			label = isPage ? __( 'Preview page' ) : __( 'Preview post' );
+		}
```

Comment:
In theory, any post type could be "viewable" depending on the post object params. See: https://github.com/ramonjd/wordpress-develop/blob/6ed1ed2e3f031e3c3f38130cd736ef80c25db5e7/src/wp-includes/post.php#L2383-L2383

But since the translated label is going to refer to either post or page only, should we rather test for post type and limit it to `post` or `page`?

E.g., `if ( postType === 'post' || postType === 'page' ) {`

I guess that if custom post types support is desired then it can be a follow up.

Reply:
I think we can consider this once we handle custom post types.

Please let me know if this we need to handle.

Reply:
I was just saying that, for this PR, we could simply test for `post` or `page`. 

I think that would be a good first version and would satisfy the reporting issue https://github.com/WordPress/gutenberg/issues/55806

Handling custom post types can always be done later I think.

Thanks!

Reply:
I'm happy to retest this once this PR is rebased and the above changes are made. Thank you!

Reply:
@ramonjd Above mentioned change is done now!

Reply:
I appreciate the speedy work! 🙇🏻